hs_core/tests/utils/test_utils.py

```diff
@@ -48,6 +49,9 @@ def wrapper(*args, **kwargs):
 
 
 def set_quota_usage_over_hard_limit(uquota, qmsg):
-    used_in_units = convert_file_size_to_unit(uquota.used_value, uquota.unit, 'B')
-    uquota.allocated_value = used_in_units / (qmsg.hard_limit_percent + 5) * 100
-    uquota.save()
+    uquota.save_allocated_value(1, "B")
+    sleep(30)
+
+
+def wait_for_quota_update(user_quota, initial_value=None, timeout=360):
```

Comment:
Thinking I should update this to set the quota to the default 20 GB and then wait